app/components/forms/orders/attendee-list.js

```diff
@@ -17,20 +23,24 @@ export default class AttendeeList extends Component {
     return this.data.user;
   }
 
+  @tracked nativeLanguage = nativeLanguage;
+  @tracked fluentLanguage = fluentLanguage;
+  @tracked genders = genders;
+
   @computed('data.attendees')
   get holders() {
     this.data.attendees.forEach(attendee => {
       if (attendee.native_language) {
-        this.languageFormMapCodeToName(attendee, 'native_language', languageForms1);
+        this.languageFormMapCodeToName(attendee, 'native_language', this.nativeLanguage);
       }
       if (attendee.fluent_language) {
-        this.languageFormMapCodeToName(attendee, 'fluent_language', languageForms2);
+        this.languageFormMapCodeToName(attendee, 'fluent_language', this.fluentLanguage);
       }
       if (attendee.language_form_1) {
-        this.languageFormMapCodeToName(attendee, 'language_form_1', languageForms1);
+        this.languageFormMapCodeToName(attendee, 'language_form_1', this.nativeLanguage);
```

Comment:
what's with language_form_1 and language_form_2?

Reply:
https://github.com/fossasia/open-event-frontend/issues/8866

For this ticket. We change the fieldIdentifier from language_form_1 to the new fieldIdentifier native_language.
However, some data was saved with language_form_1. So we also need to accommodate them

Reply:
https://github.com/fossasia/open-event-frontend/issues/8866

For this ticket. We change the fieldIdentifier from language_form_1 to the new fieldIdentifier native_language.
However, some data was saved with language_form_1. So we also need to accommodate them